include/spm.rst

```diff
@@ -3,16 +3,25 @@
 Secure Partition Manager (SPM)
 ##############################
 
-The Secure Partition Manager (SPM) uses the SPU peripheral to configure security attributions for the flash, SRAM, and peripherals.
-Note that the SPU peripheral is the nRF version of an IDAU (Implementation-Defined Security Attribution Unit).
+The Secure Partition Manager (SPM) provides functionality for the Secure Zone of the nRF9160.
+
+Overview
+********
 
-It is used in the :ref:`secure_partition_manager` sample.
+The Cortex-M33 CPU in the nRF9160 implements ARM TrustZone, which means it can run a "secure" and a "non-secure" app side by side.
+The SPM, being the secure app, takes care of configuring the permissions and resources of the non-secure app and then booting it.
```

Comment:
is responsible for configuring...

Reply:
shall we also say that it is exposing also a Secure API for the services that may be called by the NS firmware?

Reply:
Good point, added.